sdk/hatchery/register.go

```diff
@@ -229,6 +229,7 @@ func workerRegister(h Interface, models []sdk.Model) error {
 			log.Warning("workerRegister> Too many errors on spawn with model %s, please check this worker model", m.Name)
 			continue
 		}
+		log.Debug("workerRegistrer> need registration %v", h.NeedRegistration(&m))
```

Comment:
two calls of h.NeedRegistration(&m) just for debug  -> add log in if statement below